rl_loop.py

```diff
@@ -49,20 +60,48 @@ def print_flags():
                     for flag, value in flags.items()))
 
 
+def get_models():
```

Comment:
merge? I thought this was already added?

Reply:
This branch is based off of #74, but the diffbase is master, so it's not actually added.  Sorry, i know it makes the diffs hard.

Reply:
Can you rebase this commit onto a separate branch off master? This has changes for selfplay workers which I think are useful on their own and should be merged in, regardless of what we do with #74.

Reply:
Done, thanks, PTAL :)